src/templates/_macros/feed_item.html

```diff
@@ -126,33 +126,34 @@ <h1 class="name">{{ feed_app.app.name|translate(feed_app) }}</h1>
 {% macro collection_promo(coll) %}
   {% set color = coll.color or 'sapphire' %}
   {% set inline_color = feed.COLLECTION_COLORS[color] %}
-  {% set desc = coll.description|translate(coll) %}
+  {% set desc = coll.description|translate(coll)|safe %}
+  {% set has_background = True if coll.background_image %}
```

Comment:
How come we need an alias? coll.background_image will be truthy/falsy whether it has it or not